src/AppInstallerCLICore/PackageCollection.cpp

```diff
@@ -157,8 +160,19 @@ namespace AppInstaller::CLI
 
         std::optional<PackageCollection> TryParseJson(const Json::Value& root)
         {
-            // TODO: Embed schema in binaries & validate file. This will return nullopt on failure.
+            // Validate the JSON against the schema.
+            Json::Value schemaJson = JsonSchema::LoadResourceAsSchemaDoc(MAKEINTRESOURCE(IDX_PACKAGES_SCHEMA_V1), MAKEINTRESOURCE(PACKAGESSCHEMA_RESOURCE_TYPE));
+            valijson::Schema schema;
+            JsonSchema::PopulateSchema(schemaJson, schema);
+
+            valijson::ValidationResults results;
+            if (!JsonSchema::Validate(schema, root, results))
+            {
+                AICLI_LOG(CLI, Error, << JsonSchema::GetErrorStringFromResults(results));
```

Comment:
>GetErrorStringFromResults [](start = 53, length = 25)

We need to get this value to the user so that they can immediately see the issue.

Reply:
Would it be okay to show that even if it is not localized?

Reply:
I was going to add in my comment "And put one more thing on the list that needs localization."  But yes, I think even if everything else was localized, I would think it better to show a descriptive error in English (I assume) than simply an error code.  At least then the user can throw it at an automated translation and get some kind of understanding as to what is wrong (maybe).  And we can have our own localized preamble before the unlocalized part.